src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6098,6 +6098,11 @@ namespace UnitarySystems {
                     thisSys.m_HeatingSAFMethod = state.dataUnitarySystems->None;
                     if (thisSys.m_HeatCoilExists && thisSys.m_MaxHeatAirVolFlow == 0) {
                         ShowSevereError(state, cCurrentModuleObject + " = " + thisObjectName);
+                        if (loc_m_HeatingSAFMethod == "") {
```

Comment:
In the future, using `string.clear()` instead of comparing `string == ""` is preferred, but definitely not worth holding this up.